README.md

```diff
@@ -76,21 +76,14 @@ Note: If you leave out `-Dconfig.file`, the default configuration file under `sr
 
 ## 3. To define your own tests
 
-In order to add your own tests you first need to understand the core design pattern of the test-runner project.
+The simplest way to add a new test is to include it in one of the existing test suites. All tests are defined in `test-runner/` as [ScalaTest](http://www.scalatest.org/) *Spec* suites.
 
-The sbt tool manages the initial task run for the two different available running
-modes for the suite itself: the DCOS mode and the local mode (the latter means on your machine enabled by docker).
+### Add a test to an existing suite
 
-For local mode for example there is an sbt task named mit used to run the tests in local mode.
-Then this sbt task when run it creates a MesosIntegrationTestRunner instance
-which then calls different runners for different deploy modes for the spark
-itself (like cluster,client, etc). Then these runners run individual specs like
-ClientModeSpec which contain the actual test code (other specific scala test
-based specs) and which finally are submitted to a mesos cluster via a SparkJobRunner instance.
+Locate the right place to add your test. Note that currently there are two *root* suites: `ClientModeSpec` and `ClusterModeSpec`, each one mixing-in other traits with actual tests. The right place would be inside one of the traits that are mixed-in.
```

Comment:
I think the multi-dispatcher runner still ends up calling the `ClusterModeSpec`:

https://github.com/typesafehub/mesos-spark-integration-tests/blob/master/test-runner/src/main/scala/com/typesafe/spark/test/mesos/framework/runners/MultiClusterModeRunner.scala#L101